### PR TITLE
Add an API entry

### PR DESCRIPTION
**Pull request by andrewfluck**

### CODE REVIEW

**Comment by andrewfluck** [comment]
Where is the JSON failing with Travis CI???

File: graveyard.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "dateClose": "2015-12-21",
+    "dateOpen": "2005-09-14",
+    "description": "Google Blog Search API was a way to search blogs utilizing Google.",
+    "link": "https://en.wikipedia.org/wiki/Google_Blog_Search",
+    "Google Blog Search API"
```
Comment (codyogden) [inline graveyard.json]:
```suggestion
    "name": "Google Blog Search API"
```
Looks like the `name` key was missed here.

Comment (andrewfluck) [reply graveyard.json]: Oh whoops, didn't catch that, and nice ,you are using that new suggestion feature!

Comment (andrewfluck) [reply graveyard.json]: Thank you

**Comment by codyogden** [comment]
Thank you!! 💯 🙌 🥇

**Pull request merged**